README.md

```diff
@@ -18,7 +18,7 @@ In fact, to run tests or basically anything with Django management commands, you
 
 ### Environmental Settings
 
-A sample development environment settings file is in `.environ/.env.dev`. You can copy this over to your root as `.env` to use if you'd like to run `make tests` outside of a Docker container, for example.
+A sample development environment settings file is in `.environ/.env.dev`. You can copy this over to your root as `.env` to use if you'd like to run `make tests` outside of a Docker container, for example. It can also be used to override the default environmental variables passed into docker in `docker-compose.yml`.
```

Comment:
Yeah this is confusing. Both Docker and Django give `.env` priority, so that's what I should have said -- `.env` has priority over `docker-compose.yml`. I will change the statement in the earlier comment.